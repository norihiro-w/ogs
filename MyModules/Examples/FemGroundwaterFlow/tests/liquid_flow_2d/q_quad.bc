GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  LIQUID_FLOW
 $PRIMARY_VARIABLE
  PRESSURE1
 $GEO_TYPE
  POLYLINE BC
 $DIS_TYPE
  CONSTANT 1e+6
#STOP


