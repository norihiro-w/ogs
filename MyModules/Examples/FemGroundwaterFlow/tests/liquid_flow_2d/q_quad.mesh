288 
1 5 29 28 
5 6 30 29 
6 7 31 30 
7 8 32 31 
8 9 33 32 
9 10 34 33 
10 11 35 34 
11 12 36 35 
12 13 37 36 
13 14 38 37 
14 15 39 38 
15 16 40 39 
16 17 41 40 
17 18 42 41 
18 19 43 42 
19 20 44 43 
20 21 45 44 
21 22 46 45 
22 23 47 46 
23 24 48 47 
24 25 49 48 
25 26 50 49 
26 27 51 50 
27 2 52 51 
28 29 54 53 
29 30 55 54 
30 31 56 55 
31 32 57 56 
32 33 58 57 
33 34 59 58 
34 35 60 59 
35 36 61 60 
36 37 62 61 
37 38 63 62 
38 39 64 63 
39 40 65 64 
40 41 66 65 
41 42 67 66 
42 43 68 67 
43 44 69 68 
44 45 70 69 
45 46 71 70 
46 47 72 71 
47 48 73 72 
48 49 74 73 
49 50 75 74 
50 51 76 75 
51 52 77 76 
53 54 79 78 
54 55 80 79 
55 56 81 80 
56 57 82 81 
57 58 83 82 
58 59 84 83 
59 60 85 84 
60 61 86 85 
61 62 87 86 
62 63 88 87 
63 64 89 88 
64 65 90 89 
65 66 91 90 
66 67 92 91 
67 68 93 92 
68 69 94 93 
69 70 95 94 
70 71 96 95 
71 72 97 96 
72 73 98 97 
73 74 99 98 
74 75 100 99 
75 76 101 100 
76 77 102 101 
78 79 104 103 
79 80 105 104 
80 81 106 105 
81 82 107 106 
82 83 108 107 
83 84 109 108 
84 85 110 109 
85 86 111 110 
86 87 112 111 
87 88 113 112 
88 89 114 113 
89 90 115 114 
90 91 116 115 
91 92 117 116 
92 93 118 117 
93 94 119 118 
94 95 120 119 
95 96 121 120 
96 97 122 121 
97 98 123 122 
98 99 124 123 
99 100 125 124 
100 101 126 125 
101 102 127 126 
103 104 129 128 
104 105 130 129 
105 106 131 130 
106 107 132 131 
107 108 133 132 
108 109 134 133 
109 110 135 134 
110 111 136 135 
111 112 137 136 
112 113 138 137 
113 114 139 138 
114 115 140 139 
115 116 141 140 
116 117 142 141 
117 118 143 142 
118 119 144 143 
119 120 145 144 
120 121 146 145 
121 122 147 146 
122 123 148 147 
123 124 149 148 
124 125 150 149 
125 126 151 150 
126 127 152 151 
128 129 154 153 
129 130 155 154 
130 131 156 155 
131 132 157 156 
132 133 158 157 
133 134 159 158 
134 135 160 159 
135 136 161 160 
136 137 162 161 
137 138 163 162 
138 139 164 163 
139 140 165 164 
140 141 166 165 
141 142 167 166 
142 143 168 167 
143 144 169 168 
144 145 170 169 
145 146 171 170 
146 147 172 171 
147 148 173 172 
148 149 174 173 
149 150 175 174 
150 151 176 175 
151 152 177 176 
153 154 179 178 
154 155 180 179 
155 156 181 180 
156 157 182 181 
157 158 183 182 
158 159 184 183 
159 160 185 184 
160 161 186 185 
161 162 187 186 
162 163 188 187 
163 164 189 188 
164 165 190 189 
165 166 191 190 
166 167 192 191 
167 168 193 192 
168 169 194 193 
169 170 195 194 
170 171 196 195 
171 172 197 196 
172 173 198 197 
173 174 199 198 
174 175 200 199 
175 176 201 200 
176 177 202 201 
178 179 204 203 
179 180 205 204 
180 181 206 205 
181 182 207 206 
182 183 208 207 
183 184 209 208 
184 185 210 209 
185 186 211 210 
186 187 212 211 
187 188 213 212 
188 189 214 213 
189 190 215 214 
190 191 216 215 
191 192 217 216 
192 193 218 217 
193 194 219 218 
194 195 220 219 
195 196 221 220 
196 197 222 221 
197 198 223 222 
198 199 224 223 
199 200 225 224 
200 201 226 225 
201 202 227 226 
203 204 229 228 
204 205 230 229 
205 206 231 230 
206 207 232 231 
207 208 233 232 
208 209 234 233 
209 210 235 234 
210 211 236 235 
211 212 237 236 
212 213 238 237 
213 214 239 238 
214 215 240 239 
215 216 241 240 
216 217 242 241 
217 218 243 242 
218 219 244 243 
219 220 245 244 
220 221 246 245 
221 222 247 246 
222 223 248 247 
223 224 249 248 
224 225 250 249 
225 226 251 250 
226 227 252 251 
228 229 254 253 
229 230 255 254 
230 231 256 255 
231 232 257 256 
232 233 258 257 
233 234 259 258 
234 235 260 259 
235 236 261 260 
236 237 262 261 
237 238 263 262 
238 239 264 263 
239 240 265 264 
240 241 266 265 
241 242 267 266 
242 243 268 267 
243 244 269 268 
244 245 270 269 
245 246 271 270 
246 247 272 271 
247 248 273 272 
248 249 274 273 
249 250 275 274 
250 251 276 275 
251 252 277 276 
253 254 279 278 
254 255 280 279 
255 256 281 280 
256 257 282 281 
257 258 283 282 
258 259 284 283 
259 260 285 284 
260 261 286 285 
261 262 287 286 
262 263 288 287 
263 264 289 288 
264 265 290 289 
265 266 291 290 
266 267 292 291 
267 268 293 292 
268 269 294 293 
269 270 295 294 
270 271 296 295 
271 272 297 296 
272 273 298 297 
273 274 299 298 
274 275 300 299 
275 276 301 300 
276 277 302 301 
278 279 303 4 
279 280 304 303 
280 281 305 304 
281 282 306 305 
282 283 307 306 
283 284 308 307 
284 285 309 308 
285 286 310 309 
286 287 311 310 
287 288 312 311 
288 289 313 312 
289 290 314 313 
290 291 315 314 
291 292 316 315 
292 293 317 316 
293 294 318 317 
294 295 319 318 
295 296 320 319 
296 297 321 320 
297 298 322 321 
298 299 323 322 
299 300 324 323 
300 301 325 324 
301 302 3 325 
