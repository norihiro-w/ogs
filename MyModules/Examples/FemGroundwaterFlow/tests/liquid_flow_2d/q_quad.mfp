GeoSys-MAT-FP: Fluid Properties ------------------------------------
#FLUID_PROPERTIES
 $FLUID_TYPE
  LIQUID
 $DENSITY
  1 1.000000e+003
 $VISCOSITY
  1 1.000000e-003
#STOP