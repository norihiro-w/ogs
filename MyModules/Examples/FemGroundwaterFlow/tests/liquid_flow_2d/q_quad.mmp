GeoSys-MMP  Material Medium Properties ------------------------------------
#MEDIUM_PROPERTIES
 $GEOMETRY_DIMENSION
  2
 $GEOMETRY_AREA
  1.0
 $GEO_TYPE
  DOMAIN
 $POROSITY
  1  0.2
 $STORAGE
  1  1e-10
 $PERMEABILITY_TENSOR
  ISOTROPIC  1e-14
#STOP


