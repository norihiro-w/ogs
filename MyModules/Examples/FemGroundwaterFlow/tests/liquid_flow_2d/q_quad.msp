GeoSys-MSP: Material Solid Properties ---------------------------------------
#SOLID_PROPERTIES
  $DENSITY
    1 2600
  $THERMAL
    EXPANSION 
    6.0e-6
    CAPACITY 
    1 850
    CONDUCTIVITY 
    1  3.0
#STOP
