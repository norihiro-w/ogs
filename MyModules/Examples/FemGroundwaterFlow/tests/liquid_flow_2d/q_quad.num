GeoSys-NUM: Numerical Parameter ----------------------------------------
#NUMERICS
 $PCS_TYPE
 LIQUID_FLOW
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      1 1.e-14       1000           1.0   0     2
;  5      1 1.e-10       1000           1.0   100     2
 $NON_LINEAR_SOLVER 
; method error_tolerance max_iterations relaxation
  PICARD 1e-10            1            0.0
;  NEWTON 1e-10 1e-10            25            0.0
;  SNES 1e-10            1            0.0
#STOP

  UNCONFINED_FLOW
