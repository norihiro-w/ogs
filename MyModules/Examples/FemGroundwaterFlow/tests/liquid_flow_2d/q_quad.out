GeoSys-OUT: Output ------------------------------------------------
#OUTPUT
 $NOD_VALUES
  PRESSURE1
 $ELE_VALUES
  VELOCITY1
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1
#STOP

